sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2DataFrameSuite.scala

```diff
@@ -43,6 +43,7 @@ class DataSourceV2DataFrameSuite
   before {
     spark.conf.set("spark.sql.catalog.testcat", classOf[InMemoryTableCatalog].getName)
     spark.conf.set("spark.sql.catalog.testcat2", classOf[InMemoryTableCatalog].getName)
+    spark.conf.set(SQLConf.ANSI_ENABLED.key, "true")
```

Comment:
This works but we shouldn't set this conf repeatedly for each test case. I still prefer to override `def sparkConf` to set the conf once for the entire suite.